tests/benchmark/test_benchmark_row_claim.py

```
@@ -0,0 +1,293 @@
+    claim = _valid_claim()
+    claim["planner_mode"] = "degraded"
+    with pytest.raises(BenchmarkRowClaimError, match="degraded"):
+        validate_benchmark_row_claim(claim)
+
+
+def test_superlative_wording_on_diagnostic_tier_is_rejected() -> None:
+    """Superlative/ranking wording is only allowed for paper-facing successful rows."""
+    claim = _valid_claim()
+    claim["evidence_tier"] = "diagnostic"
+    claim["claim_wording"] = "Planner outperforms all baselines on this suite."
+    with pytest.raises(BenchmarkRowClaimError, match="outperforms|superlative|ranking"):
```

Ruff (0.15.17)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)